compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -2457,7 +2457,7 @@ declare_lint! {
     Deny,
     "a feature gate that doesn't break dependent crates",
```

Comment:
Should the docs be updated as well?

Reply:
Why? This still doesn't break dependent crates.

Though I do admit I find this description a bit confusing. It's not a feature gate, not really?